easybuild/easyconfigs/m/mkl_fft/mkl_fft-1.0.14-intel-2018b-Python-3.6.6.eb

```
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'mkl_fft'
+version = '1.0.14'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/IntelPython/mkl_fft'
+description = """NumPy-based Python interface to Intel(R) MKL FFT functionality"""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+source_urls = ['https://github.com/IntelPython/mkl_fft/archive']
```

The `source_urls` can be simplified by using `GITHUB_SOURCE`
```suggestion
github_account = 'IntelPython'
source_urls = [GITHUB_SOURCE]
```